Topic3/src/org/maugiam/tdd/main/RecentFileList.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * 
+ */
+package org.maugiam.tdd.main;
+
+import java.util.ArrayList;

```

Comment:
Warning: ArrayList is not used here, I guess you tried it first and then realized that methods like the push were not available. LinkedList was the correct choice to use :).